src/ucp/api/ucp.h

```diff
@@ -1036,6 +1049,12 @@ typedef struct ucp_context_attr {
      * please see @ref ucs_memory_type_t.
      */
     uint64_t              memory_types;
+
+    /**
+     * The name is intended for identification of context during tracing and
+     * analysis of UCX-based application.
+     */
+    char                  *name;
```

Comment:
which buffer size should the user allocate here?

Reply:
I will add `UCP_CONTEXT_NAME_MAX` to ucp_def.h, and change `char *name` to `char name[UCP_CONTEXT_NAME_MAX]`.